packages/features/form-builder/Components.tsx

```diff
@@ -306,7 +306,7 @@ export const Components: Record<FieldType, Component> = {
                 value.push("");
                 setValue(value);
               }}
-              className="mr-auto">
+              className="mr-auto h-fit whitespace-normal text-left">
```

Comment:
This is not how it should be fixed for email; you can simply use break-all. Also, this issue is visible in other booking fields, not just in the 'Add Guests' field

Reply:
![Screenshot 2024-09-11 001427](https://github.com/user-attachments/assets/90083a62-9d68-4886-b43f-53676f50e2d5)
It's a problem in my end

Reply:
@anikdhabal : This is happening because it is a single word. Please use multiple words and try. In addition to above classes will add work-break all aswell to break a single long word.